ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -238,12 +251,20 @@ <h3> Generated Resources:</h3>
             </li>
         </span>
     </div>
+    <div *ngIf="isDataprocJobsGenerated">
+        <span *ngIf="selectedMigrationType ==='dproc' && dataprocJobsGenerated.DataprocJobUrls != null && dataprocJobsGenerated.DataprocJobUrls.length > 0">
+            <h4>Dataproc Jobs: </h4>
+            <ul *ngFor="let url of dataprocJobsGenerated.DataprocJobUrls; let i = index">
+                <li><a [href]="url" target="_blank">{{dataprocJobsGenerated.DataprocJobIds[i]}}</a></li>
+            </ul>
+        </span>
+    </div>
     <div class="migrate">
-        <span *ngIf="!isLowDtMigrationRunning">
+        <span *ngIf="!isLowDtMigrationRunning && !isDprocMigrationRunning">
             <button mat-raised-button type="submit" color="primary" (click)="migrate()"
-                [disabled]="!((isTargetDetailSet && (selectedMigrationType ==='lowdt') && isSourceConnectionProfileSet && isTargetConnectionProfileSet) || (isTargetDetailSet && (selectedMigrationType ==='bulk'))) || isMigrationInProgress">Migrate</button>
+                [disabled]="!((isTargetDetailSet && (selectedMigrationType ==='lowdt') && isSourceConnectionProfileSet && isTargetConnectionProfileSet) || (isTargetDetailSet && (selectedMigrationType ==='bulk')) || (isTargetDetailSet && (selectedMigrationType === 'dproc') && isDataprocConfigurationSet) || isMigrationInProgress)">Migrate</button>
         </span>
-        <span *ngIf="isLowDtMigrationRunning">
+        <span *ngIf="isLowDtMigrationRunning || isDprocMigrationRunning">
```

Comment:
why are we showing end migration button in case of dataproc , there is no specific handling for dataproc migration type in endMigration () method in the ts file

Reply:
Removed
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/6e88d25c184e1071a365d97f58ae39dbf28c7702